druid/src/widget/radio.rs

```diff
@@ -14,14 +14,16 @@
 
 //! A radio button widget.
 
-use crate::kurbo::{Circle, Point, Rect, Size};
+use crate::kurbo::{Circle, Size};
 use crate::theme;
-use crate::widget::{CrossAxisAlignment, Flex, Label, LabelText, Padding};
+use crate::widget::{CrossAxisAlignment, Flex, Label, LabelText};
 use crate::{
     BoxConstraints, Data, Env, Event, EventCtx, LayoutCtx, LifeCycle, LifeCycleCtx, LinearGradient,
-    PaintCtx, RenderContext, UnitPoint, UpdateCtx, Widget, WidgetExt, WidgetPod,
+    PaintCtx, RenderContext, UnitPoint, UpdateCtx, Widget,
 };
 
+const DEFAULT_RADIO_RADIUS: f64 = 7.0;
+const INNER_CIRCLE_RADIUS: f64 = 2.0;
```

Comment:
These are consts? Is it reasonable to make them keys in the Env? Or fields? 
Right now, if I customize BASIC_WIDGET_HEIGHT, I get a centered circle based on these consts - if I make it larger, my circle doesn't grow. If I make it smaller, there's a point at which the circle overflows the bounds. I do see that it's probably unlikely to get so small, but it's certainly possible.

Reply:
I'm avoiding making any interventions in how this is actually drawn; these were previously defined in `paint` and I just moved them to the top-level. I think the desired behaviour here is up for discussion; I don't think it's necessarily wrong to say that these have a fixed size they will draw regardless of the layout size, which is similar to how text works. I can see the other side as well. Any changes to that behaviour would be a separate PR and discussion, though?

Reply:
I think I missed them being extant in the old paint code - moving them to a more visible scope is definitely a good thing. But it seems like putting them in the Env accords with the BASIC_WIDGET_SIZE and TEXT_SIZE_NORMAL being Env keys. I'll consider writing up an issue over this and some of the other bits and bobs we discussed in the Zulip to see if there's some discussion to be had.